components/navbar/sidebar.snip.html

```diff
@@ -17,15 +17,15 @@
 }}
 
 <!-- Start Sidebar -->
-<amp-sidebar id="{{sidebar-id}}" class="ampstart-sidebar px3 {{^sidebaronly}}md-hide lg-hide{{/sidebaronly}}" layout="nodisplay">
+<amp-sidebar id="{{sidebar-id}}" class="ampstart-sidebar px3 {{^sidebaronly}}md-hide lg-hide{{/sidebaronly}} {{sidebarclasses}}" layout="nodisplay">
```

Comment:
The template you are building is an opnionated template - in which case a slightly larger css is ok. The whole point of this template is to override classes in the baseline - if there is none then use bass css